arrow/locales.py

```diff
@@ -2087,6 +2018,12 @@ class BrazilianPortugueseLocale(PortugueseLocale):
     day_abbreviations = ["", "Seg", "Ter", "Qua", "Qui", "Sex", "Sab", "Dom"]
 
 
+class BrazilianPortugueseLocale(PortugueseLocale):
+    names = ["pt_br"]
+
+    past = "faz {0}"
```

Comment:
Great, if we just add a test for this one difference we should be good to go.

Reply:
Done.